docs/proposals/11-scheduling.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Reservation-Based Scheduling
+
+oep-number: 11
+
+creation-date: 2024-09-30
+
+status: draft
+
+authors:
+
+- "@lukasfrank"
+
+
+reviewers:
+
+- "@afritzler"
+- "@balpert89"
+
+---
+
+# OEP-11: Reservation-Based Scheduling
+
+## Table of Contents
+
+- [Summary](#summary)
+- [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+- [Alternatives](#alternatives)
+
+## Summary
+Scheduling resources is a central process to increase utilization of cloud infrastructure. It not only involves the process to find any feasible `pool` but rather, if possible, to find an optimal compute `pool`.
```

Comment:
Instead of refering to a general `pool` I would pin this down to `MachinePools` only.

Reply:
Also, what does "optimal" mean in this context? Should we rather say here that we find a `MachinePool` which can fullfil the resource request of a `Machine` and is able to materialize a `Machine`.